src/lib/components/tablev2/Tablev2.component.js

```diff
@@ -97,12 +116,19 @@ function Table({
     useRowSelect,
   );
 
+  useEffect(() => {
+    if (globalFilter != undefined) {
```

Comment:
```suggestion
    if (globalFilter) {
```

Reply:
if `globalFilter === ''` it should get into the if,
so i can leave it like this or for more expicit version : 
`globalFilter !== undefined && globalFilter !== null`

WDYT @JBWatenbergScality  ?

Reply:
Let's go for the explicit version `globalFilter !== undefined && globalFilter !== null `